### ISSUE TITLE
Improve the Paste from Office behaviour while using Text part language 

### ISSUE THREAD

**Issue by Mgsy** [ckeditor/ckeditor5#12672]
## 📝 Provide detailed reproduction steps (if any)

1.  Copy the [example Word document](https://ckeditor.com/docs/ckeditor5/latest/assets/CKEditor5.PFO.Sample.Recognition_of_Achievement.docx) content.
2.  Paste it to the editor with [Text part language](https://ckeditor.com/docs/ckeditor5/latest/features/language.html) and [Paste from Office](https://ckeditor.com/docs/ckeditor5/latest/features/pasting/paste-from-word.html) plugins (e.g. all-features manual test).

### ✔️ Expected result

The content doesn't have an italic style.

### ❌ Actual result

The content has an italic style.

![](https://user-images.githubusercontent.com/20988892/196408534-7a585002-735b-41d3-88d8-b3522e5d0d6a.png)

## 📃 Other details

MS Word wraps each text in `<span lang="">` so our Text part language feature kicks in, preserves the content and adds `font-style:italic` property.

![](https://user-images.githubusercontent.com/20988892/196408106-78598515-734c-4387-99f7-27833c1f61c1.png)

Translates to:

```
<p class=MsoNormal><span lang=EN-US>test<o:p></o:p></span></p>
```

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by jimmyhoeEm2ai** [comment]
👍

**Comment by Reinmar** [comment]
If anyone's wondering how to change the language in MSWord, the option is in the bottom bar:

![](https://user-images.githubusercontent.com/156149/223085567-1b362c40-7bc8-407e-b0a0-146614687575.png)

**Comment by Reinmar** [comment]
We are consider stripping `lang=*` from the content coming from Word. That's what CKE4 and Tiny do. However, that's inoptimal for two reasons:

*   This is actually a valuable information that we're losing. E.g. in a mix language content, the information about the language can help with e.g. spellcheck.
*   We should not strip the `dir` attribute because it has even greater impact on the browser behavior.

We'll need to do a bit better research regarding the optimal aproach.

**Comment by Witoso** [comment]
tl;dr after a bit of a research., I think the main task here is just to remove the italics style on the text in other languages ([Text part language](https://ckeditor.com/docs/ckeditor5/latest/features/language.html) plugin). 

To the points mentioned above, in our model's language attribute, we keep a pair in a form of: `lang:dir` (e.g. `ar:rtl`. `es:ltr`) so we shouldn't strip values.

Taking it from the user's perspective:

1.  The editor is enabled with the text part and paste plugins (I expect those features to work)
2.  I paste the content in multiple languages from Word .
3.  I have multiple languages in my editor.

To this, we add no. 4: Languages have italics style to distinguish them. I feel this is a wrong assumption. Not only do we style in italics but we also prevent removing italics which may result in bigger confusion and breaks a bit WYSIWYG idea.

To refer to Word/Pages, those tools do language autodetection but don't distinguish them in the editor UI. In Word's case, you see language in the bottom toolbar, and Pages is not showing it to you at all.

In the CKEditor case language distinction will be visible in the configured toolbar which is in my opinion enough.

**Comment by Witoso** [comment]
Let's remove the italics and add to the docs information on how to style it by yourself.
Minor BC.
